src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -62,7 +63,15 @@ public StoreController(@NotNull final StoreService storeService) {
   @Override
   public ResponseEntity<Void> storeProduct(
       final String acceptVersion, @Valid final MultipartFile file) {
-    // TODO validate Accept-Version
+    final String expectedAcceptVersion = storeApiVersion;
+    if (!StringUtils.equals(acceptVersion, expectedAcceptVersion)) {
```

Comment:
❓ Just for my understanding, a client will send to this service the expected Store API that it is trying to use?

Reply:
We should support backwards compatibility and non-major version changes, but right now we only support exactly one version. I'm not sure how this should be addressed going forward.

Reply:
Yeah, I believe it's in the header.

Reply:
Would the load balancer determine, based on this version, which service to send it to? Or to spin up a service of that version if one does not exist?

Reply:
🤷‍♀ I'm not sure how that works, but that sounds right to me.